doc/Example.md

```diff
@@ -0,0 +1,125 @@
+# Example
+
+A basic DSP Module + Tester might look like this:
+
+```scala
+package SimpleDsp
+
+// Allows you to use Chisel Module, Bundle, etc.
+import chisel3._
+// Allows you to use FixedPoint
+import chisel3.experimental.FixedPoint
+// If you want to take advantage of type classes >> Data:RealBits (i.e. pass in FixedPoint or DspReal)
+// Required for you to use operators defined via type classes (+ has special Dsp overflow behavior, etc.)
+import dsptools.numbers._
+// Enables you to set DspContext's for things like overflow behavior, rounding modes, etc.
+import dsptools.DspContext
+// Use DspTester, specify options for testing (i.e. expect tolerances on fixed point, etc.)
+import dsptools.{DspTester, DspTesterOptionsManager, DspTesterOptions}
+// Allows you to modify default Chisel tester behavior (note that DspTester is a special version of Chisel tester)
+import iotesters.TesterOptions
+// Scala unit testing style
+import org.scalatest.{FlatSpec, Matchers}
+
+// IO Bundle. Note that when you parameterize the bundle, you may need to override cloneType.
+// This also creates x, y, z inputs/outputs (direction must be specified at some IO hierarchy level)
+// of the type you specify via gen (must be Data:RealBits = UInt, SInt, FixedPoint, DspReal)
+class SimpleDspIo[T <: Data:RealBits](gen: T) extends Bundle {
+  val x = Input(gen.cloneType)
+  val y = Input(gen.cloneType)
+  val z = Output(gen.cloneType)
```

Comment:
I'm not completely sure but I think Input and Output will call clonetype for you
```suggestion
  val x = Input(gen)
  val y = Input(gen)
  val z = Output(gen)
```

Reply:
I think you're right